tests/flow/test_bulk_insertion.py

```diff
@@ -29,7 +28,7 @@ def __init__(self):
         redis_graph = Graph("graph", redis_con)
 
     def tearDown(self):
```

Comment:
I did a bit of digging on this myself today, and `tearDown` is invoked after every test - `tearDownClass` is not supported by RLTest. I think we might want to augment RLTest with this if we want to use a flush-then-sleep kludge.